Game of Game of Life.py

```diff
@@ -70,14 +69,33 @@ def check_user_input(paused):
                 paused = False
             else:
                 paused = True
+        if event.type == pygame.QUIT or pygame.key.get_pressed()[pygame.K_ESCAPE]:
+            pygame.quit()
+            import sys
+            sys.exit(0)
+        if pygame.key.get_pressed()[pygame.K_1]:
+            board = preset.place(board, 1, a, b)
+        if pygame.key.get_pressed()[pygame.K_2]:
+            board = preset.place(board, 2, a, b)
+        if pygame.key.get_pressed()[pygame.K_3]:
```

Comment:
make this into one line of code with https://www.pygame.org/docs/ref/key.html#pygame.key.name

Reply:
nice spot, went from 14 lines to 3.